src-plugins/itkDataSHImage/management/vtkSphericalHarmonicSource.cxx

```diff
@@ -42,7 +42,7 @@ double sphLegendre(const int l,const int m,const double theta) {
     const unsigned lmm = l-m;
     const unsigned lpm = l+m;
     const double fact   = boost::math::factorial<double>(lmm)/boost::math::factorial<double>(lpm);
-    const double factor = static_cast<double>(2*l+1)/(4.0*vtkMath::DoublePi())*fact;
+    const double factor = static_cast(2_l+1)/(8.0_vtkMath::Pi())*fact;
```

Comment:
2_l should be 2*l and 8.0_vtkMath :Pi() 8.0*vtkMath:Pi()

Reply:
2 * l  and 8.0 * vtkMath:Pi()

Reply:
and <double> is missing  after static cast

Reply:
< double >

Reply:
Sorry, thx, it's done